data/json/bionics.json

```
@@ -382,7 +376,8 @@
     "type": "bionic",
     "name": { "str": "Ethanol Burner" },
     "description": "You burn alcohol as fuel in an extremely efficient reaction.",
-    "occupied_bodyparts": [ [ "torso", 26 ] ],
+    "occupied_bodyparts": [ [ "torso", 16 ] ],
+    "encumbrance": [ [ "torso", 2 ] ],
     "fuel_options": [ "alcohol" ],
     "fuel_capacity": 500,
```

```suggestion
    "occupied_bodyparts": [ [ "torso", 20 ] ],
    "fuel_options": [ "alcohol" ],
    "fuel_capacity": 400,
```
It isn't sticking out, and can easily be flexible, so I wouldn't give it encumbrance. OTOH, storage does take up space.